src/components/point-prelevement/point-tabs.js

```diff
@@ -0,0 +1,24 @@
+'use client'
+
+import {Tab, Tabs, useMediaQuery} from '@mui/material'
+import {useTheme} from '@mui/material/styles'
+
+const PointTabs = ({selectedTab, handleSelectedTab}) => {
+  const theme = useTheme()
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'))
+
+  return (
+    <Tabs
+      value={selectedTab}
+      variant={isMobile ? 'scrollable' : 'standard'}
+      scrollButtons={isMobile ? 'auto' : 'off'}
```

Comment:
J'ai l'impression que le comportement est le même. C'est quoi le comportement en plus ?
```suggestion
      variant='scrollable'
      scrollButtons='auto'
```